### PR TITLE
fix: Show message & top bar when Producer opens on a processing revision

### PR DESCRIPTION
**Pull request by MarquisLP**
This fixes the issue where launching Producer on a processing revision shows only a loading spinner with no 'processing' message or any other controls.  
It also fixes a minor bug where loading another revision would cause the top bar to briefly disappear.

The cause was that we were using a single 'loading' variable for two separate statuses:

- The content object is not yet loaded (only happens at startup)
- The selected revision is not yet loaded

By creating two separate 'loading' variables, we can allow the 'processing' message and the top bar to be shown after the content has loaded, but before the selected revision has finished processing and loading.

### CODE REVIEW

File: capture/d2l-capture-producer/d2l-capture-producer.js [hunk 1]
```diff
@@ -176,16 +176,18 @@ class CaptureProducer extends RtlMixin(InternalLocalizeMixin(LitElement)) {
 	}
 
 	render() {
-		const isLoading = !this._selectedRevision || !this._src;
-		const isProcessing = !isLoading && !this._selectedRevision.draft && !this._selectedRevision.ready;
-		const isProcessingFailed = !isLoading && this._selectedRevision.processingFailed;
-		const isReadyForEdit = [isLoading, isProcessing, isProcessingFailed].every(statusFlag => !statusFlag);
+		const isLoadingContent = !this._content || !this._revisionsLatestToOldest;
```
Comment (MichaelKovalchuk) [inline capture/d2l-capture-producer/d2l-capture-producer.js:179]: if we get revisionLatestToOldest we get from the content? can we simply put: 
`const isLoadingRevisions = !this._revisionsLatestToOldest` ?

File: capture/d2l-capture-producer/d2l-capture-producer.js [hunk 2]
```diff
@@ -176,16 +176,18 @@ class CaptureProducer extends RtlMixin(InternalLocalizeMixin(LitElement)) {
...
+		const isProcessing = !isLoadingContent && !this._selectedRevision.draft && !this._selectedRevision.ready;
```
Comment (MichaelKovalchuk) [inline capture/d2l-capture-producer/d2l-capture-producer.js:180]: we are not using _selectedRevisionIsProcessing

**Comment by MichaelKovalchuk** [comment]
hi Mark, I used your PR as a platform for fixing Firefox preview bug here: https://github.com/Brightspace/content-components/pull/170

**Comment by MarquisLP** [comment]
> hi Mark, I used your PR as a platform for fixing Firefox preview bug here: #170

Cool, thanks for letting me know. I'll close this PR.

**Pull request closed without merging**